arm-logic/2015-08-01-preview/swagger/logic.json

```diff
@@ -3653,8 +3634,24 @@
             "type": "object",
             "properties": {
                 "keyVault": {
-                    "$ref": "#/definitions/ResourceReference",
-                    "description": "The key vault reference."
+                    "type": "object",
```

Comment:
@pankajsn - the previous thing was correct. Why did you make this an inline model definition. If you were using the linter, then it meant you should be adding description to the definition of ResourceReference object rather than defining it along with the $ref.